lib/rules/jsx-no-literals.js

```diff
@@ -17,6 +20,8 @@ module.exports = {
     },
 
     schema: [{
+      enum: [NO_STRINGS]
```

Comment:
Rather than add a string option, let's add this to the options object, for future extensibility.